pypeit/spectrographs/p200_dbsp.py

```diff
@@ -250,7 +254,7 @@ def default_pypeit_par(cls):
         par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
         # Change the wavelength calibration method
         par['calibrations']['wavelengths']['method'] = 'full_template'
-        par['calibrations']['wavelengths']['lamps'] = ['FeI', 'FeII', 'ArI', 'ArII']
+        par['calibrations']['wavelengths']['lamps'] = ['FeAr']
```

Comment:
Is there a good reason for combining these?  I have always considered it optimal to have them ion by ion.  Please explain

Reply:
The IRAF FeAr line list had some lines that were prominent in DBSPb arcs that got tossed out by cutting NIST lines on relative intensity (which looks to be how the FeI, FeII, ArI and ArII line lists got created). I can put the FeAr lines back into the ion by ion lists if desired.

Reply:
Yes, I do desire it.  Sorry for the trouble.

Reply:
No worries, I have added the lines to FeI, ArI, and ArII ion line lists.